README.md

```diff
@@ -53,13 +144,13 @@ For example:
 cmake_minimum_required(VERSION 3.5)
 project(test_example VERSION 1.0.0)
 
-if(WASM_ROOT STREQUAL "" OR NOT WASM_ROOT)
-    set(WASM_ROOT ${CMAKE_INSTALL_PREFIX})
+if(EOSIO_WASMSDK_ROOT STREQUAL "" OR NOT EOSIO_WASMSDK_ROOT)
+    set(EOSIO_WASMSDK_ROOT "/usr/local/eosio.wasmsdk")
 endif()
 list(APPEND CMAKE_MODULE_PATH ${WASM_ROOT}/lib/cmake)
```

Comment:
`${WASM_ROOT}/lib/cmake` here should instead be `${EOSIO_WASMSDK_ROOT}/lib/cmake`, right?